pkg/controller/certificates/certificate_controller_test.go

```diff
@@ -37,7 +37,7 @@ func TestCertificateController(t *testing.T) {
 	}
 
 	client := fake.NewSimpleClientset(csr)
-	informerFactory := informers.NewSharedInformerFactory(client, controller.NoResyncPeriodFunc())
+	informerFactory := informers.NewSharedInformerFactory(fake.NewSimpleClientset(csr), controller.NoResyncPeriodFunc())
```

Comment:
Because we don't care about interactions that the informer makes against the client. We only want to make assertions against the interactions taken by the handle.